aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -595,6 +595,22 @@ Tensor hann_window(
       window_length, periodic, /*alpha=*/0.5, /*beta=*/0.5, options);
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ conversions ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tensor cpu(const Tensor& self) {
+  return self.to(kCPU);
+}
+
+Tensor cuda(const Tensor& self, int64_t device_id, bool non_blocking) {
+  if (device_id == -1) { // default argument
+    return self.to(kCUDA, non_blocking);
+  } else {
+    return self.to({kCUDA, device_id}, non_blocking);
+  }
+}
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I don't know why I didn't put `~~~ tensor ~~~` above here before, but please write `tensor` where there is currently a space